src/js/components/NameValuePair/NameValuePair.js

```diff
@@ -25,7 +25,21 @@ const NameValuePair = ({ children, name: nameProp }) => {
   let name;
   if (typeof nameProp === 'string' || typeof nameProp === 'number')
     name = (
-      <Text as="dt" textAlign={nameAlign} {...theme.nameValuePair.name}>
+      <Text
+        as="dt"
+        margin={
+          column
```

Comment:
I'm wondering if we should check the theme too. Something feels a bit odd about passing `margin={{ bottom: undefined }}`.